platform/src/main/java/com/neueda/jetbrains/plugin/graphdb/platform/SupportedLanguage.java

```diff
@@ -0,0 +1,26 @@
+package com.neueda.jetbrains.plugin.graphdb.platform;
+
+import java.util.stream.Stream;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public enum SupportedLanguage {
+    CYPHER("Cypher");
+
+    private final String languageId;
+
+    SupportedLanguage(String languageId) {
+        this.languageId = languageId;
+    }
+
+    public String getLanguageId() {
+        return languageId;
+    }
+
+    public static boolean isSupported(String languageId) {
+        checkNotNull(languageId, "'languageId' is undefined");
```

Comment:
minor: check on null is redundant here.

Reply:
I don't agree with that. Why do we need do something if value languageId is null? In this case if languageId is null then something wen't wrong before.